mne/channels/_standard_montage_utils.py

```diff
@@ -334,3 +334,46 @@ def _read_brainvision(fname, head_size):
         pos *= head_size / np.median(np.linalg.norm(pos, axis=1))
 
     return make_dig_montage(ch_pos=_check_dupes_odict(ch_names, pos))
+
+
+def _read_csv(fname, delimiter=','):
+    """Import eeg channel locations from CSV files into MNE instance.
+
+    CSV files should have columns x, y, and z, each row represents one channel.
+    Optionally the first column can contain the channel names.
+
+    Parameters
+    ----------
+    fname : str
+        Name of the csv file to read channel locations from
+    delimiter : str
+        Delimiter used by the CSV file
+    include_ch_names : bool
+        Whether the CSV file include channel names as the first column
+    """
+    include_ch_names = True
+    import csv
```

Comment:
python standard library imports should go top-of-file, not nested inside function. We only nest optional dependencies (or internal imports to avoid circular import errors)